package.json

```diff
@@ -27,7 +27,7 @@
     "tmp": "~0.0.25"
   },
   "optionalDependencies": {
-    "phantomjs": "~1.9.7-1"
+    "phantomjs": "^2.1.2"
```

Comment:
I got this warning:
npm WARN deprecated phantomjs@2.1.3: Package renamed to phantomjs-prebuilt. Please update 'phantomjs' package references to 'phantomjs-prebuilt'